web/src/main/webapp/WEB-INF/jsp/include/showTableForEventDefinition.jsp

```diff
@@ -155,7 +157,7 @@
 				<tr>
 					<c:set var="isFirstLink" value="${true}" />
 					<c:if test="${table.filtered}">
-						<td class="table_tools"><a href="<c:out value="${removeFilterQuery}"/>">Clear Search Keywords</a></td>
+						<td class="table_tools"><a href="<c:out value="${removeFilterQuery}"/>"><fmt:message key="clear_search_keywords" bundle="${restext}"/></a></td>
 						<c:set var="isFirstLink" value="${false}" />
```

Comment:
This key doesn't capitalize all words like in the original text.  Need a new property file entry.